### PR TITLE
Restore windows CI

### PR DESCRIPTION
**Pull request by CodyCBakerPhD**
Restoring the previous state of attempting to fix Windows pipelines

There also seems to be a new failure in the Windows DevTests suite (but never BuildTests) following merge of https://github.com/NeurodataWithoutBorders/nwb-guide/pull/786

At time of merge it was only seen in the LiveServices tests (and thus disregarded as an impersistent live service specific problem) and not in the dev tests; but it's been seen in the DevTests for Windows ever since as well

### CODE REVIEW

**Comment by garrettmflynn** [comment]
Fixed the Pipelines test on Windows! Was making this too complicated by adding an `.env` file, you can just directly set the environment variable using Github Actions.

As for the DevTests failure, I'm still not sure what's going on there...

**Comment by CodyCBakerPhD** [comment]
Incredible - admirable persistence on this issue!

What do you think about disabling dev tests for Windows on this PR and continuing debugging for that on yet another follow-up?

File: .github/workflows/testing_pipelines.yml [hunk 1]
```diff
@@ -95,33 +95,18 @@ jobs:
...
-        run: type .env
-        shell: bash
-        if: runner.os == 'Windows'
-
       # Run pipeline tests
       - if: matrix.os != 'ubuntu-latest'
         name: Run tests
         run: npm run test:pipelines
+        env:
+          GIN_DATA_DIRECTORY: ${{ github.workspace }}
```
Comment (CodyCBakerPhD) [inline .github/workflows/testing_pipelines.yml:103]: Ohhh, I had no idea these were (or could be) scoped per line under `run`

**Comment by garrettmflynn** [comment]
Sure sounds good to me.

**Comment by CodyCBakerPhD** [comment]
Cool - this PR technically needs your approval since I'm the one who opened it yesterday 😆

**Review by garrettmflynn — APPROVED** [review]
Gotcha. Looks good!

**Pull request merged**